src/doxyfile

```diff
@@ -813,7 +813,7 @@ RECURSIVE              = YES
 # Note that relative paths are relative to the directory from which doxygen is
 # run.
 
-EXCLUDE                = solvers/z3/
+EXCLUDE                = solvers/z3/ ../doc/cprover-manual
```

Comment:
I'd personally follow the pattern used for other list variables in this file, and separate the two paths by a backslash and new line pair. I think that makes it more readable.

Reply:
👍